x-pack/plugins/observability/public/components/shared/exploratory_view/series_builder/columns/report_breakdowns.test.tsx

```diff
@@ -0,0 +1,75 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0; you may not use this file except in compliance with the Elastic License
+ * 2.0.
+ */
+
+import React from 'react';
+import { fireEvent, screen } from '@testing-library/react';
+import { render } from '../../../../../utils/test_helper';
+import { getDefaultConfigs } from '../../configurations/default_configs';
+import { mockIndexPattern, mockUrlStorage } from '../../rtl_helpers';
+import { NEW_SERIES_KEY } from '../../hooks/use_url_strorage';
+import { ReportBreakdowns } from './report_breakdowns';
+import { USER_AGENT_OS } from '../../configurations/data/elasticsearch_fieldnames';
+
+describe('Series Builder ReportBreakdowns', function () {
+  const dataViewSeries = getDefaultConfigs({
+    reportType: 'pld',
+    indexPattern: mockIndexPattern,
+    seriesId: NEW_SERIES_KEY,
+  });
+
+  it('should render properly', function () {
+    mockUrlStorage({});
+
+    render(<ReportBreakdowns dataViewSeries={dataViewSeries} />);
+
+    screen.getByText('Select an option: , is selected');
+    screen.getAllByText('Browser family');
+  });
+
+  it('should set new series breakdown on change', function () {
+    const { setSeries } = mockUrlStorage({});
+
+    render(<ReportBreakdowns dataViewSeries={dataViewSeries} />);
+
+    const btn = screen.getByRole('button', {
+      name: /select an option: Browser family , is selected/i,
+      hidden: true,
+    });
+
+    fireEvent.click(btn);
+
+    fireEvent.click(screen.getByText(/operating system/i));
+
+    expect(setSeries).toHaveBeenCalledTimes(1);
+    expect(setSeries).toHaveBeenCalledWith('newSeriesKey', {
+      breakdown: USER_AGENT_OS,
+      reportType: 'pld',
+      time: { from: 'now-15m', to: 'now' },
+    });
+  });
+  it('should set undefined on new series on no select breakdown', function () {
+    const { setSeries } = mockUrlStorage({});
+
+    render(<ReportBreakdowns dataViewSeries={dataViewSeries} />);
+
+    const btn = screen.getByRole('button', {
```

Comment:
Why use screen instead of returning getByRole from the render function? Just wondering.

Reply:
i actually use chrome extension to copy selector for rtl it provides with `screen.` so that's why.